packages/perseus-editor/src/__docs__/editor-page-with-storybook-preview.tsx

```diff
@@ -26,6 +31,23 @@ type Props = {
     hints?: ReadonlyArray<Hint>;
 };
 
+const testDependenciesV2: PerseusDependenciesV2 = {
```

Comment:
Can you import the one from [here](https://github.com/Khan/perseus/blob/5d2a6b07ef29fdbd6b3458b8abe4dbad480ce2b5/testing/test-dependencies.tsx#L104) instead of creating a new object here?